fr/firefox_cloud_services_tos.md

```diff
@@ -1,42 +1,42 @@
-﻿# Services cloud de Firefox : Conditions d’utilisation
+﻿# Mozilla - Conditions d’utilisation
```

Comment:
This is not for the whole Mozilla, it's for Mozilla accounts alone.

Again, stop gap solution: `Comptes Mozilla - Conditions d’utilisation`, but check with the translator.